tests/nightly/test_large_vector.py

```diff
@@ -715,12 +715,13 @@ def test_full():
 
 
 def test_one_hot():
-    a = nd.zeros(10)
-    a[0] = 1
-    a[-1] = 1
+    #default dtype of ndarray is float32 which cannot index elements over 2^32
+    a = nd.array([1], dtype=np.int64)
```

Comment:
maybe you don't even need this array a?

just do `a = nd.array([1], LARGE_X)`

Reply:
`a = nd.array([1, LARGE_X])` would result in b.shape = 2 x 4300000000.
keeping `a` as single element helps keeps b.shape = 1 x 4300000000.
@apeforest do you agree ?

Reply:
it still expects numpy array and errors out `AssertionError: Argument indices must have NDArray type` cannot avoid that